lib/src/fields/form_builder_filter_chips.dart

```diff
@@ -149,10 +149,10 @@ class FormBuilderFilterChip extends FormBuilderField<dynamic> {
         );
 
   @override
-  _FormBuilderFilterChipState createState() => _FormBuilderFilterChipState();
+  _FormBuilderFilterChipState<T> createState() => _FormBuilderFilterChipState();
```

Comment:
Without my IDE, I'm not 100% sure, but...  Do you need to say:
```dart
  _FormBuilderFilterChipState<T> createState() => _FormBuilderFilterChipState<T>();
```
Do you need the `<T>` on the right side too?

Reply:
It is not necessary, I have already removed it.